lib/MusicBrainz/Script/Role/IncrementalDump.pm

```diff
@@ -433,7 +433,7 @@ sub get_current_replication_sequence {
     my $replication_info_uri = $self->replication_access_uri . '/replication-info';
     my $response = $c->lwp->get("$replication_info_uri?token=" . DBDefs->REPLICATION_ACCESS_TOKEN);
 
-    unless ($response->code == 200) {
+    unless ($response->code == HTTP_OK) {
```

Comment:
I wrote an awk script to check the diff for this commit (so, if a removed line contained "200", then the next added line should contain "HTTP_OK', etc. for each code/name pair defined in https://metacpan.org/pod/HTTP::Status#CONSTANTS). It didn't work in cases where the line was broken up into multiple, but it turned out that those were the only ones I had to check (other than a couple false positives). Everything else was apparently fine.